.azure-pipelines/baseline_test/baseline.test.template.yml

```diff
@@ -98,3 +105,5 @@ jobs:
           STOP_ON_FAILURE: "False"
           TEST_PLAN_NUM: $(BASELINE_MGMT_PUBLIC_MASTER_TEST_NUM)
           NUM_ASIC: $(NUM_ASIC)
+          VM_TYPE: $(VM_TYPE)
+          SPECIFIC_PARAM: $(SPECIFIC_PARAM)
```

Comment:
Does this parameter have a default value? Such as in t1-lag checker?

Reply:
Will add